### PR TITLE
hide tooltip on navigation and fix undo

### PR DESCRIPTION
**Pull request by MartinRyan**
hide tooltips on navigating away from the graph view and fix undo

### CODE REVIEW

File: src/views/Graph.vue [hunk 1]
```diff
@@ -504,7 +505,7 @@ export default {
       }
 
       async function registerExtensions () {
-        // register extensions
+        // // register extensions
```
Comment (kinow) [inline src/views/Graph.vue]: accidental double comment?

**Comment by kinow** [comment]
Tested `master` first. Clicking on `Graph`, then clicking on a node, it displays a popup. By clicking on another link like `Dashboard`, now the new view is loaded, but the popup is still being displayed.

Also, while in the `Graph` view, moving nodes around, CTRL+Z does nothing on `master`.

Happy to report this PR indeed fixes both issues :clap: and I had no idea we had an undo feature. Thanks @MartinRyan !

File: src/views/Graph.vue (hunk 1 above)
Comment (MartinRyan) [reply src/views/Graph.vue]: lo yep!

**Comment by hjoliver** [comment]
@MartinRyan - did you accidentally merge this?

**Comment by kinow** [comment]
@hjoliver, maybe when @MartinRyan left for vacation we were still doing the 1 review only? I think I asked in Riot about that around the time he was leaving or had left already...

**Comment by hjoliver** [comment]
@kinow - I was wondering that too, but @MartinRyan also merged this without fixing the double comment you noted above ... so I thought maybe it was an accident!  (Also it isn't "customary" to merge your own PRs!).  (No biggie, just trying to figure out what the situation is).

**Pull request merged**